.github/workflows/cd-syftcli.yml

```diff
@@ -0,0 +1,91 @@
+name: CD - Syft-CLI
+
+on:
+  schedule:
+    - cron: "00 10 * * */7" # At 10:00 UTC on every three days
+
+  workflow_dispatch:
+    inputs:
+      skip_tests:
+        description: "If true, skip tests"
+        required: false
+        default: "false"
+
+jobs:
+  call-pr-tests-linting:
+    if: github.repository == 'OpenMined/PySyft' && (github.event.inputs.skip_tests == 'false' || github.event_name == 'schedule') # don't run on forks
+    uses: OpenMined/PySyft/.github/workflows/pr-tests-linting.yml@dev
+
+  # call-pr-tests-syft:
+  #   if: github.repository == 'OpenMined/PySyft' && (github.event.inputs.skip_tests == 'false' || github.event_name == 'schedule') # don't run on forks
+  #   uses: OpenMined/PySyft/.github/workflows/pr-tests-syft.yml@dev
+
+  # call-pr-tests-stack:
+  #   if: github.repository == 'OpenMined/PySyft' && (github.event.inputs.skip_tests == 'false' || github.event_name == 'schedule') # don't run on forks
+  #   uses: OpenMined/PySyft/.github/workflows/pr-tests-stack.yml@dev
+  #   secrets: inherit
+
+  deploy-syft-cli:
+    needs: [call-pr-tests-linting]
+    if: always() && (needs.call-pr-tests-linting.result == 'success' || github.event.inputs.skip_tests == 'true')
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.SYFT_BUMP_TOKEN  }}
+      - name: Install checksumdir
+        run: |
+          pip install --upgrade checksumdir
+      - name: Get the hashes
+        id: get-hashes
+        shell: bash
+        run: |
+          current_hash=$(checksumdir ./packages/syft-cli)
+          echo "current_hash=$current_hash" >> $GITHUB_OUTPUT
+          previous_hash=$(cat ./scripts/syft-cli_hash)
+          echo "previous_hash=$previous_hash" >> $GITHUB_OUTPUT
+
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.11"
+
+      - name: Install dependencies
+        if: ${{steps.get-hashes.outputs.current_hash}} != ${{steps.get-hashes.outputs.previous_hash}}
+        run: |
+          python -m pip install --upgrade pip
+          pip install --upgrade tox setuptools wheel twine bump2version PyYAML
+
+      - name: Bump the Version
+        if: ${{steps.get-hashes.outputs.current_hash}} != ${{steps.get-hashes.outputs.previous_hash}}
+        run: |
+          python3 syft-cli/version.py
```

Comment:
NITPICK: With the current path changes, I think, this would be 

```python3
python3 src/syftcli/version.py
```

I think also we need to add a print statement, when we execute the script directly
in version.py

```python3
if __name__ == "__main__":
    print(__version__)
```

like we do in hagrid, as we currently do not print.